packages/ember-application/lib/system/application.js

```diff
@@ -316,46 +295,19 @@ var Application = Namespace.extend(DeferredMixin, {
     Create a container for the current application's registry.
 
     @private
-    @method buildContainer
+    @method buildInstance
     @return {Ember.Container} the configured container
   */
-  buildContainer: function() {
+  buildInstance: function() {
     var container = this.__container__ = this.__registry__.container();
```

Comment:
Yes, @wycats and I discussed this a bit and felt that, as much as this fact might pain us, the `__container__` API is de facto public and removing it would be very annoying to a great many people. Given that it is not particularly onerous to continue supporting it, at least until 2.0, we thought it best to preserve it.